lib/exe/fetch.php

```diff
@@ -55,6 +55,15 @@
         'status'        => $STATUS,
         'statusmessage' => $STATUSMESSAGE,
         'ispublic'      => media_ispublic($MEDIA),
+        'csp' => [
+            'sandbox' => '',
+            'default-src' => "'none'",
+            'script-src' => "'none'",
```

Comment:
because `default-src` is the fallback for many (but not all) and has the same value `script-src` could be omitted